### DISCUSSION TITLE
Route53 GetHostedZoneLimit

### DISCUSSION THREAD

**Discussion by Fruan-Atlassian** [aws/aws-sdk-kotlin#1312] [General]
Just wondering whether the Route53 GetHostedZoneLimit request is working for anyone? For me, I can't use `getHostedZoneLimitRequest`'s `type` parameter as I get an error saying that it's not of enum `MAX_RRSETS_BY_ZONE` when I use `HostedZoneLimitType.MaxRrsetsByZone`.
Is anyone able to do this on their end?

**Comment by ianbotsf** [comment]
I'm able to reproduce this too and it seems like a bug. Filed https://github.com/awslabs/aws-sdk-kotlin/issues/1314 to track a fix.

**Reply by Fruan-Atlassian** [reply]
Cheers mate! Thanks for the confirmation!

**Reply by ianbotsf** [reply]
The fix for this has been merged and should be part of today's SDK release **v1.2.22**.

**Reply by Fruan-Atlassian** [reply]
That's great! Thanks a lot for that!
